sdk/formrecognizer/azure-ai-formrecognizer/tests/test_receipt.py

```diff
@@ -421,3 +421,32 @@ def test_receipt_continuation_token(self, client):
         result = poller.result()
         self.assertIsNotNone(result)
         initial_poller.wait()  # necessary so azure-devtools doesn't throw assertion error
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    def test_receipt_locale_default(self, client):
+        def _get_locale_in_call(pipeline_response, _, headers):
+            assert 'en-US' == pipeline_response.http_response.request.query['locale']
```

Comment:
I'm trying to understand this. So we are doing a call and we want to see if the service returned `en-US`? or does we set a default in the SDK>

Reply:
basically, I'm trying to confirm if the user doesn't pass in `locale`, the default locale that is being used is in the call to the service is `en-US`. While this checks more the swagger / our generation of it, I wanted to include it since we are saying in our docstring that the default is `en-US`

Reply:
actually @maririos i realized based off of your comment that my tests aren't actually working, since the default isn't set in the swagger, it's set on the service side. so going to redo my checking of these tests

Reply:
but we do want to set it in the SDK as default, right? just verifying

Reply:
no, we should leave that up to the service. Ignore everything I said about swagger, since it's not actually set as the default in the swagger. The service is saying (in their docs at least): that if you don't send us `locale` in your request in a query parameter, we're going to use our default of 'en-US'

Reply:
My test actually isn't correct, because my callback where I'm checkign the query parameters in the call to the service actually is never called, so please ignore this test 😆

Reply:
This goes against what Java is doing. Before merging this PR let's make sure that our SDKs are aligned 
FYI @samvaity

Reply:
@maririos @samvaity merging bc paul has p much decided this for us 😄